integration-tests/smoke/ccip/ccip_migration_to_v_1_6_test.go

```diff
@@ -96,9 +96,46 @@ func TestV1_5_Message_RMNRemote(t *testing.T) {
 	require.NoError(t, err)
 	oldState, err := changeset.LoadOnchainState(e.Env)
 	require.NoError(t, err)
-	e = testhelpers.AddCCIPContractsToEnvironment(t, e.Env.AllChainSelectors(), tEnv, false)
+	envNodes, err := deployment.NodeInfo(e.Env.NodeIDs, e.Env.Offchain)
```

Comment:
The code looks very similar across all the tests, can we extract this into a func w/ an appropriate name?